src/helpers/clearAccessToken.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import { serialize as serializeCookie } from 'cookie'
+import { AUTH_COOKIE } from '@constants/cookies'
+
+const clearAccessToken = (context: any): void => {
+  const epochTimestamp = Math.floor(new Date() / 1000)
+  const cookies = serializeCookie(AUTH_COOKIE, '', {
+    // Setting a `maxAge` in the past expires the cookie.
+    maxAge: epochTimestamp
+  })
+
+  if (context && context.ctx.res) {
+    const response = context.ctx.res
+    response.setHeader('Set-Cookie', cookies)
```

Comment:
We don’t actually clear the token server-side but I guess we could want to do that at some point